CadRevealComposer/CadRevealComposerRunner.cs

```diff
@@ -84,6 +86,9 @@ IReadOnlyList<IModelFormatProvider> modelFormatProviders
             // collect all nodes for later sector division of the entire scene
             nodesToExport.AddRange(cadRevealNodes);
 
+            // Todo should this return a new list of cadrevealnodes instead of mutating the input?
```

Comment:
Any reason to not just return a new list?

Reply:
Maybe to reduce memory usage? Suggest moving a possible refactoring to its own task?